netplan_cli/cli/commands/apply.py

```diff
@@ -332,21 +333,19 @@ def command_apply(self, run_generate=True, sync=False, exit_on_error=True, state
                 utils.nm_bring_interface_up(loopback_connection)
 
     @staticmethod
-    def is_composite_member(composites, phy):
+    def is_composite_member(netdef: NetDefinition) -> bool:
```

Comment:
todo: looks like this method is not used any more (besides in its own tests). Let's drop it then?